cmake/Reprojucer.cmake

```diff
@@ -1781,7 +1812,11 @@ function(__generate_AppConfig_header)
     __bool_to_int("${JUCER_BUILD_AUDIOUNIT_V3}" Build_AUv3_value)
     __bool_to_int("OFF" Build_RTAS_value) # Not yet supported
     __bool_to_int("OFF" Build_AAX_value) # Not yet supported
-    __bool_to_int("${JUCER_BUILD_AUDIOUNIT_V3}" Build_STANDALONE_value)
+    if(DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)
```

Comment:
Would it make sense to define a global variable somewhere with this expression? You're checking it in quite a few places now.

Reply:
I think there are already too many global variables, so I want to avoid adding more if possible. I could write
```
if(DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)
  set(juce4 TRUE)
else()
  set(juce4 FALSE)
endif()
```
in functions that use that condition many times, but I won't do it in this PR.

Reply:
Sounds reasonable!